platform/brains2/src/duo/main.cpp

```diff
@@ -132,7 +135,7 @@ static void pots_read() {
   synth.delay = pinRead(DELAY_PIN);
 }
 
-static void power_off() {
+static inline void power_off() {
```

Comment:
`inline` doesn't really do anything here (and in general doesn't since it's just a hint).
But, a static function like this will be inlined whenever suitable regardless of the `inline` keyword afaik.